pom.xml

```diff
@@ -277,6 +278,12 @@
 				<artifactId>archunit-junit5</artifactId>
 				<version>${archunit-junit5.version}</version>
 			</dependency>
+			<dependency>
+				<groupId>org.mongodb</groupId>
+				<artifactId>mongodb-driver-reactivestreams</artifactId>
+				<version>${mongodb-driver-reactivestreams.version}</version>
+				<scope>test</scope>
```

Comment:
scope is not needed in the dependency management section

Reply:
Fixed it